sdfObject/sdfobject-voltage.sdf.json

```diff
@@ -0,0 +1,103 @@
+{
+  "info": {
+    "title": "Voltage",
+    "version": "2021-10-27",
+    "copyright": "Copyright (c) 2018-2021 IPSO",
+    "license": "https://github.com/one-data-model/oneDM/blob/master/LICENSE"
+  },
+  "sdfObject": {
+    "Voltage": {
+      "label": "Voltage",
+      "description": "This object should be used with voltmeter sensor to report measured voltage between two points. It also provides resources for minimum and maximum measured values, as well as the minimum and maximum range that can be measured by the sensor. An example measurement unit is volts.",
```

Comment:
"An example measurement unit is volts." I'm not aware of any alternatives, the unit of measure is volt